### ISSUE TITLE
configure nginx to not serve .txt files directly, or at least not robots.txt file

### ISSUE THREAD

**Issue by atrandafir** [thegooddata/webapp#45]
I have setup a directory under /protected/robots with a .txt file for each domain and a Yii url rule that will forward the path /robots.txt to the .txt file corresponding to the active domain the user is visiting.

So I've got:
protected/robots/www.thegooddata.org.txt
protected/robots/pre.thegooddata.org.txt 

This way we can have multiple robots.txt files for various domains and it basically works for basic most of the environments.

The problem is that pre environment uses NGINX specific configuration that serves static files directly, so what happens is that when accesing /robots.txt, the request is not interpreted by Yii, it skips apache and php and it looks for the file locally on the disk.

So basically, what needs to be done is to make nginx not to serve TXT files directly.

This issue affects #2 , if it can't be done, the other solution would be to create a physical file on the pre server named robots.txt and ignore it from git and but keep it on every deployment, because right now on each deployment it gets deleted. We cannot just add it to git with Disallow: / because it will also affect production.
